components/Icons.es6.js

```diff
@@ -1837,6 +1837,14 @@ function Spaceship ({ className, width="16", height="16", viewBox="0 0 16 16", c
   )
 }
 
+function Zap ({ className, width='11', height='16', color="#D7D7DE", viewBox="0 0 11 16" }) {
```

Comment:
I am curious why is the width 11 and not 16 like other icons?

Reply:
Do you have screenshots to compare how both differ?

Reply:
11 to 16 requirement is defined in the source of the icon (see below)